core/src/test/java/org/apache/spark/util/collection/TestTimSort.java

```diff
@@ -110,22 +110,22 @@ private static int minRunLength(int n) {
    * @param X The sum of the sequence that should be added to runs.
    */
   private static void generateJDKWrongElem(List<Long> runs, int minRun, long X) {
-    for (long newTotal; X >= 2 * minRun + 1; X = newTotal) {
+    for (long newTotal; X >= 2L * minRun + 1; X = newTotal) {
```

Comment:
My guess is this can't actually overflow for some reason, but I think it's OK, esp in a test. These look fine.

Reply:
Thank for your review @srowen ~ Overflow may not actually occur, but I think it looks safer after change